README.md

```diff
@@ -45,32 +44,41 @@ If something goes wrong and it prints some red errors, try to add ` --user` flag
 3. Prepare your Takeout:
 
 If your Takeout was dividied into multiple `.zip`s, you will need to extract them, and move their contents into one folder. 
-
-Because I don't have good solution on how to handle albums, you will need to cut off all ["Album folders"](#why-do-you-need-to-cut-out-albums) - those who are not named like "2016-06-26" or "2016-06-26 #2" - don't worry, all photos from albums are in corresponding "date folders" already - they would just make a duplicate.
-
 Now, you should be able to just run it straight in cmd/terminal:
 
 4. `google-photos-takeout-helper -i [INPUT TAKEOUT FOLDER] -o [OUTPUT FOLDER]`
 
 // Or if this doesn't work: `python3 -m google_photos_takeout_helper -i [INPUT TAKEOUT FOLDER] -o [OUTPUT FOLDER]`
 
-// Ps note: Jezus fucking Christ people, without the "[ ]" :facepalm:
+// Ps note: Don't use the "[ ]" in the command above.
```

Comment:
I understand it's frustrating but this is pretty condescending. Especially for non-dev folk trying to look for a solution for exporting their Google stuff.

Reply:
I just really like to swear in code/comments :clown_face: 

Sure, this PR can bring some wholesomeness :sparkling_heart:

Reply:
Oh I'm all for swearing. This one just felt directed 😅.